__tests__/app.test.js

```diff
@@ -22,19 +22,71 @@ describe('GET/api/categories', () => {
             let resultArray = response.body
 
             expect(resultArray.length).toBe(4) 
-
             resultArray.forEach((category) => {
                 expect(category).toHaveProperty("slug")
                 expect(category).toHaveProperty("description")
             })
         })
     });
- 
 });
 
-describe('Deals with api paths that does not exist', () => {
+describe('Error Handler - Deals with api paths that does not exist', () => {
     test('resolves with 404: Not found, for routes that does not exist', () => {
-        return request(app).get('/invalidAPI').expect(404)
+        return request(app).get('/invalidAPI').expect(404).then((response) => {
+            expect(response.body).toEqual({msg: 'Invalid API path'})
+        })
+    })
+});
+
+describe('GET/api/reviews', () => {
+    test('should return a status code of 200', () => {
+        return request(app).get('/api/reviews').expect(200)
+    });
+    test('should return an array', () => {
+        return request(app).get('/api/reviews').then(({body}) => {
+            expect(Array.isArray(body)).toBe(true)
+        })
+    });
+    test('each element of the array should contain following properties', () => {
+        return request(app).get('/api/reviews').then(({body}) => {
+            expect(body.length).toBe(13)
+            body.forEach((review) => {
+                expect(review).toHaveProperty("owner")
```

Comment:
We could be more specific and add a second argument to .toHaveProperty(). This method can take a second argument that can either check the literal value of that property. Or it can take an assertion that can check the type of the value at this property. 

expect.any() might come in handy here - have a look at the jest docs